### PR TITLE
[EDA] Settings Saving Functionality - Front End

### PR DESCRIPTION
**Pull request by skosgi1**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: force-app/main/default/lwc/settingsSaveCanvas/settingsSaveCanvas.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<template>
+    <div class="slds-grid slds-theme_default slds-grid--align-spread">
+
+        <div class="slds-col slds-m-left_x-small">
+            <span>ComponentTitle</span>
```
Comment (MatthewBlanski) [inline force-app/main/default/lwc/settingsSaveCanvas/settingsSaveCanvas.html:5]: This should be a variable so it is inherited
```suggestion
            <span>{componentTitle}</span>
```

File: force-app/main/default/lwc/settingsSaveCanvas/settingsSaveCanvas.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        </div>
+
+        <div class="slds-col">
+            <span>
+                <div if:true={editButtonShown}>	
+                    <lightning-button label={labelReference.settingsButtonEdit} title="Edit" onclick={handleEditClick} class="slds-p-right_x-small"></lightning-button>	
+                </div>	
+                <div if:false={editButtonShown}>	
+                    <lightning-button label={labelReference.settingsButtonCancel} title="Cancel" onclick={handleCancelClick} class="slds-m-left_x-small"></lightning-button>
+                    <lightning-button variant="brand" label={labelReference.settingsButtonSave} title="Primary action" onclick={handleSaveClick} class="slds-p-right_x-small"></lightning-button>
```
Comment (MatthewBlanski) [inline force-app/main/default/lwc/settingsSaveCanvas/settingsSaveCanvas.html:15]: Let's use the custom label for the titles too. We may need more for A11y concerns.
```suggestion
                    <lightning-button label={labelReference.settingsButtonCancel} title={labelReference.settingsButtonCancel} onclick={handleCancelClick} class="slds-m-left_x-small"></lightning-button>
                    <lightning-button variant="brand" label={labelReference.settingsButtonSave} title={labelReference.settingsButtonSave} onclick={handleSaveClick} class="slds-p-right_x-small"></lightning-button>
```

File: force-app/main/default/lwc/settingsSaveCanvas/settingsSaveCanvas.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<template>
+    <div class="slds-grid slds-theme_default slds-grid--align-spread">
+
+        <div class="slds-col slds-m-left_x-small">
+            <span>{ComponentTitle}</span>
```
Comment (MatthewBlanski) [inline force-app/main/default/lwc/settingsSaveCanvas/settingsSaveCanvas.html:5]: If you can, go for the styling you see in the mock for this.

**Review by MatthewBlanski — CHANGES_REQUESTED** [review]
A few minor changes, some to do with styling, some to do with handling the way the data flows and nuances in how the overall design should take place that weren't necessarily communicated

**Review by MatthewBlanski — CHANGES_REQUESTED** [review]

**Review by MatthewBlanski — APPROVED** [review]

**Comment by spinkelman** [comment]
QA Passes - Merging

**Pull request merged**